src/components/Spinner/SpinnerActions.js

```diff
@@ -0,0 +1,2 @@
+export const SHOW_SPINNER_ACTION = 'SHOW_SPINNER_ACTION';
```

Comment:
This action is used only in `SpinnerSagas.js`. The saga exports `showSpinner()`.

So basically this action is internal for the saga, nobody is going to ever reuse it, because only `showSpinner()` is for external use.

We're going to end up with `SpinnerActions.js` being used only in `SpinnerSagas.js`. And the file has only 2 lines and I guess we're not going to add more actions for the spinner ever?

On the other looks very well structured. Maybe the structure is more important than having no 2-line files :)

If we had one big file for all the actions it would be hard to refactor (split) it later.

![show-spinner-action](https://user-images.githubusercontent.com/6686679/57652244-d0064100-75d7-11e9-9ebe-809b2445fc06.png)

Reply:
I agree with that. In this case this file a redundant and was created to avoid change structure in future. But you're right. For now it can be removed.

Reply:
removed